### DISCUSSION TITLE
Winform和Wpf

### DISCUSSION THREAD

**Discussion by SamaHddd** [BYJRK/DotNet-Discussions#202] [C# 技术交流]
想问一下Winform和Wpf的未来发展？孰优孰劣？

**Comment by BYJRK** [comment]
首先，如果你有机会不选择 WinForms，那么最好还是不要选了。除非性能对你来说是一个非常重要的东西，并且要开发的软件在功能上并不怎么复杂，也不需要拥抱诸如 MVVM 等模式。
